server/tests/employer.site.test.js

```diff
@@ -0,0 +1,248 @@
+    postalCode: 'V1V2V3',
+    postalCodeFsa: 'V1V',
+    phoneNumber: '2502223333',
+    emailAddress: 'eddy@example.com',
+    interested: 'yes',
+    nonHCAP: 'yes',
+    crcClear: 'yes',
+    preferredLocation: 'Fraser',
+  };
+
+  const participant2 = {
+    maximusId: 648691,
+    lastName: 'Finkleman',
+    firstName: 'Freduardo',
+    postalCode: 'V1V2V3',
+    postalCodeFsa: 'V1V',
+    phoneNumber: '2502223333',
+    emailAddress: 'freddy@example.com',
+    interested: 'yes',
+    nonHCAP: 'yes',
+    crcClear: 'yes',
+    preferredLocation: 'Fraser',
+  };
+
+  // Used for single site POST
+  // siteId must be assigned by the test
+  const siteBaseFields = {
+    siteName: 'Test site',
+    phaseOneAllocation: 1,
+    address: '123 XYZ',
+    city: 'Victoria',
+    healthAuthority: 'Vancouver Island',
+    postalCode: 'V8V 1M5',
+    registeredBusinessName: 'AAA',
+    operatorName: 'Test Operator',
+    operatorContactFirstName: 'AABB',
+    operatorContactLastName: 'CCC',
+    operatorEmail: 'test@hcpa.fresh',
+    operatorPhone: '2219909090',
+    siteContactFirstName: 'NNN',
+    siteContactLastName: 'PCP',
+    siteContactPhone: '2219909091',
+    siteContactEmail: 'test.site@hcpa.fresh',
+  };
+
+  // Used for batch site POST
+  const site = {
+    siteId: 67,
+    siteName: 'Test site',
+    phaseOneAllocation: 1,
+    address: '123 XYZ',
+    city: 'Victoria',
+    healthAuthority: 'Vancouver Island',
+    postalCode: 'V8V 1M5',
+    registeredBusinessName: 'AAA',
+    operatorName: 'Test Operator',
+    operatorContactFirstName: 'AABB',
+    operatorContactLastName: 'CCC',
+    operatorEmail: 'test@hcpa.fresh',
+    operatorPhone: '2219909090',
+    siteContactFirstName: 'NNN',
+    siteContactLastName: 'PCP',
+    siteContactPhoneNumber: '2219909091',
+    siteContactEmailAddress: 'test.site@hcpa.fresh',
+  };
+
+  it('Create new single site, receive success', async () => {
+    siteBaseFields.siteId = 90;
+    const res = await saveSingleSite(siteBaseFields);
+    expect(res.siteId).toEqual(90);
+    expect(res.id).toBeDefined();
+  });
+
+  it('Create duplicate single site, receive dupe', async () => {
+    siteBaseFields.siteId = 91;
+    const res = await saveSingleSite(siteBaseFields);
+    expect(res.siteId).toEqual(91);
+    expect(res.id).toBeDefined();
+
+    try {
+      await saveSingleSite(siteBaseFields);
+    } catch (excp) {
+      expect(excp.code).toEqual('23505');
+    }
+  });
+
+  it('Create new site via batch, receive success', async () => {
+    const res = await saveSites(site);
+    const expectedRes = [
+      { siteId: 67, status: 'Success' },
+    ];
+    expect(res).toEqual(expectedRes);
+  });
+
+  it('Get sites, receive all successfully', async () => {
+    const res = await getSites();
+    expect(res).toEqual(
+      expect.arrayContaining(
+        [site].map((item) => (expect.objectContaining(item))),
+      ),
+    );
+  });
+
+  it('Gets a single site', async () => {
```

Comment:
Updated this test so it creates the site it's loading, instead of assuming which site has which ID